docs/CHANGELOG.md

```diff
@@ -13,6 +13,12 @@ Entries inside each section should be ordered by type:
 ## CLI
 !-->
 
+# unreleased - YYYY-MM-DD
+
+## Python API
+
+* [Fixed] Allow S3 paths starting with `/` in `Package().set_dir()` ([#4207](https://github.com/quiltdata/quilt/pull/4207))
```

Comment:
I think this fixes `Package.set()` as well
if so might make sense to add test for it as well

Reply:
You're right!